recipes/config.rb

```diff
@@ -31,6 +31,13 @@
 syslog = node['qubit_bamboo']['syslog'] ? '2>&1 | logger -p user.info -t bamboo' : ''
 bamboo_command = "#{bin} #{flags} #{syslog}"
 
-poise_service 'bamboo-server' do
+poise_service 'bamboo' do
   command bamboo_command
 end
+
+# disable the old bamboo-server service
+service 'bamboo-server' do
+  provider Chef::Provider::Service::Upstart
+  action [:disable, :stop]
+  ignore_failure true
```

Comment:
Maybe you could add something like : 

only_if { File.exist?("/etc/init/bamboo-server.conf") }